tools/custom-checks/get-all-tgz-path.ts

```diff
@@ -0,0 +1,53 @@
+import path from "path";
```

Comment:
I think functions from this file should be merged with the mains script, it's short and not self-contained. For me, custom-checks should contain only executables

Reply:
as for this one, I got this from get-all-pkg-dirs.ts. I based the code there for get-all-tgz-path.ts so I don't know how to resolve this one